scripts/mail-alarm

```diff
@@ -34,66 +34,43 @@ FromLineOverride=YES
 ma_username="__dom0__mail_alarm"
 
 mail_language_pack_path = '/etc/xapi.d/mail-languages/'
+session = XenAPI.xapi_local()
+
 
 def log_err(err):
     print(err, file=sys.stderr)
     syslog.syslog(syslog.LOG_USER | syslog.LOG_ERR, "%s: %s" % (sys.argv[0], err))
 
-def get_pool_name():
-    session = XenAPI.xapi_local()
-    session.xenapi.login_with_password(ma_username, "", "1.0", "xen-api-scripts-mail-alarm")
-    try:
-        opaque_ref = session.xenapi.pool.get_all()[0]
-        pool_name = session.xenapi.pool.get_name_label(opaque_ref)
-        if pool_name == "":
-            master_ref = session.xenapi.pool.get_master(opaque_ref)
-            master_name = session.xenapi.host.get_name_label(master_ref)
-            return master_name
-        else:
-            return pool_name
-    finally:
-        session.xenapi.session.logout()
+def get_pool_name():  
+    opaque_ref = session.xenapi.pool.get_all()[0]
+    pool_name = session.xenapi.pool.get_name_label(opaque_ref)
+    if pool_name == "":
+        master_ref = session.xenapi.pool.get_master(opaque_ref)
+        master_name = session.xenapi.host.get_name_label(master_ref)
+        return master_name
+    else:
+        return pool_name
 
 def get_sr_name_by_uuid(sr_uuid):
-    session = XenAPI.xapi_local()
-    session.xenapi.login_with_password(ma_username, "", "1.0", "xen-api-scripts-mail-alarm")
-    try:
-        opaque_ref = session.xenapi.SR.get_by_uuid(sr_uuid)
-        sr_name = session.xenapi.SR.get_name_label(opaque_ref)
-        return sr_name
-    finally:
-        session.xenapi.session.logout()
+    opaque_ref = session.xenapi.SR.get_by_uuid(sr_uuid)
+    sr_name = session.xenapi.SR.get_name_label(opaque_ref)
+    return sr_name
 
 def get_pool_other_config():
-    session = XenAPI.xapi_local()
-    session.xenapi.login_with_password(ma_username, "", "1.0", "xen-api-scripts-mail-alarm")
-    try:
-        opaque_ref = session.xenapi.pool.get_all()[0]
-        return session.xenapi.pool.get_other_config(opaque_ref)
-    finally:
-        session.xenapi.session.logout()
+    opaque_ref = session.xenapi.pool.get_all()[0]
+    return session.xenapi.pool.get_other_config(opaque_ref)
 
 def get_VM_params(uuid):
-    session = XenAPI.xapi_local()
-    session.xenapi.login_with_password(ma_username, "", "1.0", "xen-api-scripts-mail-alarm")
     try:
-        try:
-            opaque_ref = session.xenapi.VM.get_by_uuid(uuid)
-            return session.xenapi.VM.get_record(opaque_ref)
-        finally:
```

Comment:
Indentation was wrong here, I think this code used to be broken, now its fixed.

Reply:
Indentation in the file is also wrong elsewhere. Running [black](https://black.readthedocs.io/en/stable/) on the file might a good thing to do. The changes it produces should added last as a separate commit.